### ISSUE TITLE
Selected vulkan device does not support timeline semaphores

### ISSUE THREAD

**Issue by xiatutu** [haasn/libplacebo#156]
Hello!
I got a error in ffmpeg:
![image](https://user-images.githubusercontent.com/33248045/220064943-b95e7f3d-ea55-4411-9398-9fb87b5dccb8.png)

enviroment: windows 10, vulkan sdk 1.3.239.0,  graphics driver is Intel(R) UHD Graphics 630 （I also tested on nvidia geforce RTX 2080, got the same error.）
It seems to be related to libplacebo:

const VkPhysicalDeviceTimelineSemaphoreFeatures *timeline_sem;
    timeline_sem = vk_find_struct(&vk->features,
        VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_TIMELINE_SEMAPHORE_FEATURES);
    if (!timeline_sem || !timeline_sem->timelineSemaphore) {
        PL_ERR(vk, "Selected vulkan device does not support timeline semaphores!");
        goto error;
    }

ffmpeg is built with "--enable-vulkan --enable-libplacebo". And, I run "ffmpeg -init_hw_device list", shows:
Supported hardware device types:
cuda
dxva2
qsv
d3d11va
opencl
vulkan

I have no idea to solve it, Can you give some advice?

**Comment by haasn** [comment]
Can you show your `vulkaninfo`?

**Comment by xiatutu** [comment]
> Can you show your `vulkaninfo`?

Hello, I solve it by updating the driver. 

But when I ran the cmd-"ffmpeg -i hdr.h265 -init_hw_device vulkan -vf hwupload,libplacebo=format=yuv420p,hwdownload,format=yuv420p test_hdr.h265"

 I got errors:
**Failed to allocate frame to upload to**.
Error while filtering: Cannot allocate memory
Failed to inject frame into filter network: Cannot allocate memory
Error while processing the decoded data for stream #0:0

How can I fix it? I want to use libplacebo to achieve sdr2hdr, but it seems I got an error in hwupload.
